clients/source.go

```diff
@@ -119,7 +122,9 @@ func (c *SourceClient) newManagedClient(ctx context.Context, path string) (*Sour
 	}()
 	c.cmd = cmd
 
+	c.wg.Add(1)
```

Comment:
you prob also want to to do `cq.wg.Add` before the previous `go func()` . This is why we didn't see the `plugin exited` in the tests

Reply:
But I will add it to the other one too 👍